test/create-snowpack-app/__snapshots__/create-snowpack-app.test.js.snap

```diff
@@ -7506,26 +7506,26 @@ document.head.appendChild(styleEl);"
 `;
 
 exports[`create-snowpack-app app-template-react: _dist_/App.js 1`] = `
-"import React, { useState } from '../web_modules/react.js';
-import logo from './logo.svg.proxy.js';
-import './App.css.proxy.js';
-function App() {
-  return /*#__PURE__*/React.createElement(\\"div\\", {
+"import React, {useState} from \\"../web_modules/react.js\\";
```

Comment:
we now use esbuild instead of Babel in this app, which created the output changes here